smartcontactmanager/src/main/java/com/scm/smartcontactmanager/controller/contact/ContactController.java

```diff
@@ -0,0 +1,71 @@
+package com.scm.smartcontactmanager.controller.contact;
+
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PatchMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.data.domain.PageRequest;
+
+import com.scm.smartcontactmanager.controller.contact.request.ContactRequestDto;
+import com.scm.smartcontactmanager.entity.contact.Contact;
+import com.scm.smartcontactmanager.service.contact.ContactService;
+
+
+@RestController
+public class ContactController {
+
+	@Autowired
+	ContactService contactService;
+
+	@PostMapping(value = "/v1/contacts", consumes = "application/json", produces = "application/json")
+	public ResponseEntity<String> createContact(@RequestBody ContactRequestDto contactRequestDto) {
+		contactService.createContact(contactRequestDto);
+		return new ResponseEntity<>("Succesfully created", HttpStatus.OK);
```

Comment:
Make the message clear. For ex: "Contact Successfully Created." Make sure to do this across other methods in this class

Reply:
Resolved.